src/components/Card/card.css

```diff
@@ -47,3 +48,9 @@
     width: 40px;
     @apply lil-float-right lil-h-full;
 }
+
+@screen sm {
```

Comment:
good for medium screens ?

Reply:
yep fixed h solved multiple sized cards issue.. checked for all sizes

Reply:
yep